### PR TITLE
Add researcher badge

### PR DESCRIPTION
**Pull request by ErikSchierboom**
This is the script to award the badge:

```ruby
usernames = %w[
  bergjohan
  bkhl
  bubo-py
  ceddlyburge
  coriolinus
  ee7
  ErikSchierboom
  goalaleo
  iHiD
  kntsoriano
  kytrinyx
  mmmmmrob
  mpizenberg
  nicolechalmers
  pvcarrera
  SleeplessByte
  sshine
  tehsphinx
  TheLostLambda
  yawpitch
]

User.where(github_username: usernames).find_each do |user|
  AwardBadgeJob.perform_later(user, :researcher)
end
```

### CODE REVIEW

File: app/images/icons/researcher.svg [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (ErikSchierboom) [inline app/images/icons/researcher.svg]: This is a temporary badge

File: app/models/badges/researcher_badge.rb [hunk 2]
```diff
@@ -0,0 +1,40 @@
+module Badges
+  class ResearcherBadge < Badge
+    seed "Researcher",
```
Comment (ErikSchierboom) [inline app/models/badges/researcher_badge.rb:3]: I was torn between naming this `Researcher` and naming this `Scientist`. Let me know which you prefer.

Comment (kotp) [reply app/models/badges/researcher_badge.rb:3]: How about perquisitor? or seeker?  Or tryer-outer or maybe zetetic.

File: app/models/badges/researcher_badge.rb [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+      :ultimate,
+      'researcher',
+      'Awarded for having helped with Exercism Research'
```
Comment (iHiD) [inline app/models/badges/researcher_badge.rb]:
```suggestion
      'Helped develop Exercism Research'
```

File: app/models/badges/researcher_badge.rb [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+    def award_to?(user)
+      RESEARCHERS.include?(user.handle.downcase)
```
Comment (iHiD) [inline app/models/badges/researcher_badge.rb]: Are the handles exercism handles or GH handles?

Comment (ErikSchierboom) [reply app/models/badges/researcher_badge.rb]: Let me double check

Comment (ErikSchierboom) [reply app/models/badges/researcher_badge.rb]: They're github usernames.

File: test/models/badges/researcher_badge_test.rb [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+  end
+
+  test "award_to?" do
+    badge = create :researcher_badge
+
+    non_researcher_user = create :user
+    refute badge.award_to?(non_researcher_user)
+
+    non_github_user = create :user, github_username: nil
+    refute badge.award_to?(non_github_user)
```
Comment (ErikSchierboom) [inline test/models/badges/researcher_badge_test.rb:21]: Added this test to ensure there is no nil exception when a user does not have a github username

**Pull request merged**